### ISSUE TITLE
V.1.1.19 - Music Assistant DB remains empty and seems disabled

### ISSUE THREAD

**Issue by Krzysztonek** [music-assistant/hass-music-assistant#255]
Please see the below UI screen:

![image](https://user-images.githubusercontent.com/87784133/170935942-e7e8f19f-405c-4155-9d0f-89cf94159e73.png)

No UI buttons are accessible/clickable, the DB remains empty(?) with the initial size 144 kB.
Please advise.

**Comment by erkr** [comment]
Hi,
My update is working as expected (150mb DB file). Did you do a fresh install? Are the media sources like Spotify configured? Anything in the log file?
Best Eric

**Comment by Krzysztonek** [comment]
Yes, I did a fresh install. Neither Spotify nor any other streaming option is enabled (I have no account on Spotify).
I'm not able to find Music Assistant log file... strange..

**Comment by marcelveldt** [comment]
You did not enable any of the music providers... In that case it kind of makes sense that Music Assistant is empty :-)

Or did you enable the "local files" provider ?
In that case, did you both enable the file provider and enter the path to the files ?

There is no music assistant logfile, the logging for Music Assistant is in the Home Assistant logging.

**Comment by erkr** [comment]
@Krzysztonek 
> I have no account on Spotify or any other streaming providers

You can try to add TuneIn, it's available as a free service. Two notes, use your user name (visible in the TuneIn profile, not your email) and add some radio stations as favorites to your TuneIn library.
With that MA will give you radio stations to play with.

**Comment by Krzysztonek** [comment]
@erkr ,
Indeed, all works fine for me since when I added some free streaming services. Thank you! 🍺
